content/blog/how-to-generate-microsoft-dsc-adapted-resource-manifests-with-dscresource.authoring.md

```
@@ -0,0 +1,440 @@
+    Update-DscAdaptedResourceManifest -PropertyOverride $override |
+    ForEach-Object { $_.ToJson() }
+```
+
+You can use this to perform post-processing where the generated schema needs more attention. Time to look at the final part: how can resource authors easily adopt it in their existing workflows
+
+## Automate generation in build.yaml
+
+Each DSC community module contains a `build.yaml`. To easily adopt the automatic generation of an adapted resource manifest, you can define one of the two build tasks:
+
+1. `Create_DscAdaptedResourceManifests` - which generates `*.dsc.adaptedResource.json` files for each class-based resource
+2. `Create_DscResourceManifestsList` - which generates `*.dsc.manifests.json` file containing multiple entries to adapted resources
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)